tests/utils.tpl

```diff
@@ -851,6 +853,15 @@ check_open_file_count_or_die (int lineno)
 #endif
 } /* check_open_file_count_or_die */
 
+void
+get_unique_test_name (const char ** filename, const char * test)
+{	static char	buffer [1024] ;
+
+	snprintf (buffer, sizeof (buffer) - 1, "%s_%s", test, *filename) ;
```

Comment:
@bobsayshilol foudn the bug. For the final review, could you amend this? `snprintf` assumes the maximum length it can use is `n-1`, i.e. it's already `NUL`-termination aware and doesn't require passing `n-1` into it:

```suggestion
	snprintf (buffer, sizeof (buffer), "%s_%s", test, *filename) ;
```

Reply:
Derp. Good spot!